api/src/main/java/org/pmiops/workbench/db/model/DbUserEgressBypassWindow.java

```diff
@@ -0,0 +1,109 @@
+package org.pmiops.workbench.db.model;
+
+import java.sql.Timestamp;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@Entity
+@Table(name = "user_egress_bypass_window")
+@EntityListeners(AuditingEntityListener.class)
+public class DbUserEgressBypassWindow {
+  private long egressBypassId;
+  private DbUser user;
+
+  private Timestamp startTime;
+  private Timestamp endTime;
+
+  private String description;
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "egress_bypass_id", nullable = false)
+  public long getEgressBypassId() {
+    return egressBypassId;
+  }
+
+  public DbUserEgressBypassWindow setEgressBypassId(long egressBypassId) {
+    this.egressBypassId = egressBypassId;
+    return this;
+  }
+
+  @ManyToOne
+  @JoinColumn(name = "user_id")
+  public DbUser getUser() {
+    return user;
+  }
+
+  public DbUserEgressBypassWindow setUser(DbUser user) {
+    this.user = user;
+    return this;
+  }
+
+  @Column(name = "start_time", nullable = false)
+  public Timestamp getStartTime() {
+    return startTime;
+  }
+
+  public DbUserEgressBypassWindow setStartTime(Timestamp startTime) {
+    this.startTime = startTime;
+    return this;
+  }
+
+  @Column(name = "end_time", nullable = false)
+  public Timestamp getEndTime() {
+    return endTime;
+  }
+
+  public DbUserEgressBypassWindow setEndTime(Timestamp endTime) {
+    this.endTime = endTime;
+    return this;
+  }
+
+  @Column(name = "description", nullable = false)
```

Comment:
@yonghaoy in Liquibase, nullable = true.  I think `false` is the right choice: we want to require a description, for audit reasons.